README.md

```diff
@@ -1,16 +1,18 @@
 # Reco
 
-Reco is an audio recorder focused on recording.
+Reco is a fully-featured audio recorder.
+
+You can use it to record and remember spoken words, system audio, improvized melodies, and anything else you can do with a microphone or speaker.
 
 ![Welcome view](data/screenshots/pantheon/welcome-init-light.png) ![Recording view](data/screenshots/pantheon/recording-light.png)
 
 Features include:
 
-* **Recording sounds from both your microphone and system at the same time.** This would be useful when you record conversations with chat apps or stream videos on the Internet
-* **Saving in some commonly used formats.** It supports ALAC, FLAC, MP3, Ogg Vorbis, Opus, and WAV
-* **Timed recording.** You can set a delay before recording up to 15 seconds, and the length of recording up to 600 seconds
-* **Autosaving or always-ask-where-to-save workflow.** You can select whether the app saves recordings automatically or you choose a destination each time on saving them
-* **Saving recording when the app quits.** Even if you happen to quit the app while recording, the recording is either saved automatically, or the file chooser dialog is shown - depending on your saving setting
+* **Recording sounds from both your microphone and system at the same time.** This is be useful for when you need to record calls or stream videos on the Internet.
```

Comment:
`is be`

Is this grammatically correct?

Reply:
Maybe like this?

```suggestion
* **Recording sounds from both your microphone and system at the same time.** This is useful for recording calls or streaming videos on the Internet.
```

Reply:
I missed that. Thank you for pointing that out.